### PR TITLE
Add meta infos for win 8.1 Tile application

### PR DESCRIPTION
**Pull request by ponceta**
Enhancement for win 8.1 users and future windows tiles

[Test](https://mf-geoadmin3.dev.bgdi.ch/ltpoa)

#2273

### CODE REVIEW

File: src/index.mako.html [hunk 1]
```diff
@@ -28,8 +28,11 @@
     <meta name="viewport" content="initial-scale=1.0, user-scalable=no, minimal-ui"/>
     <meta name="apple-mobile-web-app-capable" content="yes"/>
     <meta name="apple-mobile-web-app-status-bar-style" content="black"/>
+    <meta name="application-name" content="map.geo.admin.ch"/>
     <meta name="msapplication-TileImage" content="${versionslashed}img/touch-icon-bund-144x144.png"/>
     <meta name="msapplication-TileColor" content="#ffffff"/>
+    <meta name="msapplication-square70x70logo" content="${versionslashed}img/touch-icon-bund-76x76.png"/>
+    <meta name="msapplication-square150x150logo" content="${versionslashed}img/touch-icon-bund-152x152.png"/>
```
Comment (gjn) [inline src/index.mako.html]: For both images, should size is different from is size. Will it work?

Comment (ponceta) [inline src/index.mako.html]: I need testing devices ;)

**Comment by ponceta** [comment]
wp_ss_20150410_0001
White logo on win 8.1 = not MS compliant color

File: src/index.mako.html [hunk 2]
```diff
@@ -28,8 +28,11 @@
     <meta name="viewport" content="initial-scale=1.0, user-scalable=no, minimal-ui"/>
     <meta name="apple-mobile-web-app-capable" content="yes"/>
     <meta name="apple-mobile-web-app-status-bar-style" content="black"/>
+    <meta name="application-name" content="map.geo.admin.ch"/>
```
Comment (ponceta) [inline src/index.mako.html]: This should be as title - changing dynamically

Comment (oterral) [inline src/index.mako.html]: That takes the good name before so do we really need it?

Comment (ponceta) [inline src/index.mako.html]: With this, we don't "suppose" page-title will be taken as application-name.

**Comment by gjn** [comment]
@ponceta what's the state of this PR?

**Comment by ponceta** [comment]
@pauloamado @davidoesch we need a new logo for windows tiles.

On windows 7 it is great 

image

but on win phone 8.1...

image

**Comment by ponceta** [comment]
Amazon has the same problem with white logo, but at least they have their name on it.

**Comment by davidoesch** [comment]
all the logos
http://intranet.efd.admin.ch/efd/fachinfo/00016/00893/index.html?lang=de

**Comment by ponceta** [comment]
image

latest version [test](https://mf-geoadmin3.dev.bgdi.ch/ltpoa)

**Comment by gjn** [comment]
@davidoesch acceptable.

**Comment by davidoesch** [comment]
accetable

**Pull request merged**